sklearn/metrics/_scorer.py

```diff
@@ -682,34 +685,36 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
 fowlkes_mallows_scorer = make_scorer(fowlkes_mallows_score)
 
 
-SCORERS = dict(explained_variance=explained_variance_scorer,
-               r2=r2_scorer,
-               max_error=max_error_scorer,
-               neg_median_absolute_error=neg_median_absolute_error_scorer,
-               neg_mean_absolute_error=neg_mean_absolute_error_scorer,
-               neg_mean_squared_error=neg_mean_squared_error_scorer,
-               neg_mean_squared_log_error=neg_mean_squared_log_error_scorer,
-               neg_root_mean_squared_error=neg_root_mean_squared_error_scorer,
-               neg_mean_poisson_deviance=neg_mean_poisson_deviance_scorer,
-               neg_mean_gamma_deviance=neg_mean_gamma_deviance_scorer,
-               accuracy=accuracy_scorer, roc_auc=roc_auc_scorer,
-               roc_auc_ovr=roc_auc_ovr_scorer,
-               roc_auc_ovo=roc_auc_ovo_scorer,
-               roc_auc_ovr_weighted=roc_auc_ovr_weighted_scorer,
-               roc_auc_ovo_weighted=roc_auc_ovo_weighted_scorer,
-               balanced_accuracy=balanced_accuracy_scorer,
-               average_precision=average_precision_scorer,
-               neg_log_loss=neg_log_loss_scorer,
-               neg_brier_score=neg_brier_score_scorer,
-               # Cluster metrics that use supervised evaluation
-               adjusted_rand_score=adjusted_rand_scorer,
-               homogeneity_score=homogeneity_scorer,
-               completeness_score=completeness_scorer,
-               v_measure_score=v_measure_scorer,
-               mutual_info_score=mutual_info_scorer,
-               adjusted_mutual_info_score=adjusted_mutual_info_scorer,
-               normalized_mutual_info_score=normalized_mutual_info_scorer,
-               fowlkes_mallows_score=fowlkes_mallows_scorer)
+SCORERS = dict(
```

Comment:
Can we minimize the diff by only adding `neg_mean_absolute_percentage_error_scorer`? and not change the spacing?

Reply:
Yep, we can do that but it will give pep8 error. @ogrisel suggested me to do like this [here](https://github.com/scikit-learn/scikit-learn/pull/15007#discussion_r382100880).

Reply:
Maybe it's fine to break PEP8 for this one. Using 2 spaces indent is not great either. Sorry about the back and forth.

You can put a `  # noqa` comment to silence the flake8 linter on that specific line.

Reply:
Done